sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsConf.scala

```diff
@@ -457,6 +457,12 @@ object RapidsConf {
     .booleanConf
     .createWithDefault(false)
 
+  val DECIMAL_TYPE_ENABLED = conf("spark.rapids.sql.decimalType.enabled")
+      .doc("Enable decimal types up to 64-bit precision. Note that not all GPU " +
```

Comment:
nit: I'm not sure that this tells the whole story. I would prefer something like.

```
.doc("Enable decimal type support on the GPU.  Decimal support on the GPU is limited to less than 18 digits and " +
    "and is only supported by a small number of operations currently.  This can result in a lot of data movement " +
    "to and from the GPU, which can slow down processing in some cases.")
```

Reply:
let me change that

Reply:
that's done @revans2